### DISCUSSION TITLE
openSUSEway default terminal app

### DISCUSSION THREAD

**Discussion by denisok** [openSUSE/openSUSEway#61] [Polls]
Hey community! 

There is proposal PR to replace `alarcritty` with `foot`:
https://github.com/openSUSE/openSUSEway/pull/57 

Lets vote and discuss!

Please suggest other alternatives :)

**Comment by m-rey** [comment]
I'd like to suggest [kitty](https://sw.kovidgoyal.net/kitty/) ([link to openSUSE package](https://software.opensuse.org/package/kitty)) as an alternative :)
I've been using it since I've started using sway and had no issues

**Reply by jubalh** [reply]
I think kitty was the old default for sway users but people moved on since then. I think the trend migration was kitty -> alacritty -> foot.

**Reply by mcepl** [reply]
I don't think kitty is a good idea. One of the nice things about sway is that it is rather minimalistic, and with all respect (and I am a professional Python developer, so I don't have much bias against Python) a terminal application written in Python with all its dependencies is not minimalist. foot seems much lighter.
